app/_assets/stylesheets/badges.less

```diff
@@ -58,25 +54,132 @@
 
   &.oss::after {
       content: "OSS ONLY";
-      opacity: 0.7;
       color: white;
       border-color: #6e30bf;
       background-color: #6e30bf;
       text-align: center;
     }
+
+
+  // Tooltip container
+  .tooltip {
+    position: relative;
+    display: inline-block;
+    margin: 0;
+    padding: 0;
+    visibility: hidden;
+  }
+
+  // Tooltip text styling
+  &:hover {
+    .tooltiptext[data-title]:after {
+      font-family: @family-display;
+      content: attr(data-title);
+      display: block;
+      font-size: 13px;
+      font-weight: 400;
+      line-height: 20px;
+      left: 0;
+      top: 100%;
+      margin-top: 0.5rem;
+      color: #fff;
+      width: 150px;
+      background-color: @black-80;
+      text-align: center;
+      padding: 5px;
+      border-radius: 4px;
+      position: absolute;
+      z-index: 10;
+      visibility: visible;
+    }
+  }
 }
 
 span.badge {
   vertical-align: text-bottom;
+}
 
-  &:after {
-    margin: 2px;
-  }
+h2.badge, h3.badge, h4.badge, h5.badge {
+    &:after {
+      margin-left: 7px;
+    }
+
+    // &:hover {
+    //   .tooltiptext[data-title]:before {
+    //     font-weight: 400;
+    //     font-size: 13px;
+    //   }
+    // }
 }
 
 a.badge {
   &:hover, &:focus {
-    opacity: 0.7;
+    opacity: 0.9;
     text-decoration: none;
   }
 }
+
+// Tooltips in tables; for the last table row, the tooltip moves to the right of
+// the element so that the table doesn't add a scrollbar
+table > tbody > tr {
+  td {
+      .badge {
+        .tooltip {
+          vertical-align: bottom;
+          right: 0;
+        }
+        &:hover {
+        .tooltiptext[data-title]:after {
+          bottom: 140%;
+          vertical-align: top;
+          left: unset;
+          margin-left: 2px;
+          margin-bottom: -5px;
+          top: unset;
+          }
+        }
+      }
+    }
+  //
+  // &:last-of-type {
```

Comment:
For whatever reason, this method doesn't work with the current approach (it worked in attempt #1). However, I was able to get around needing it by moving the tooltip above.